### PR TITLE
update story_text function to return word count

### PR DESCRIPTION
**Pull request by afroman32**
# Description
This pull request updates the submission_text function to return the word count of the submission along with everything else.
We needed to add imports to app/utils/complexity/squad_score.py:
- nltk.download('punkt')
- nltk.download('averaged_perceptron_tagger')

Pair programmed with @peterger8y @BritVandy

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: app/api/submission.py [hunk 1]
```diff
@@ -60,6 +60,11 @@ async def submission_text(sub: Submission):
     # score the transcription using SquadScore algorithm
     score = await squad_score(transcriptions, scaler)
 
+    # # takes in the story as a string
+    # # counts all words per story submission
```
Comment (BritVandy) [inline app/api/submission.py:64]: Yay! I’m so glad it worked!

File: app/utils/complexity/squad_score.py [hunk 2]
```diff
@@ -4,6 +4,9 @@
 import nltk
 import pandas as pd
 
+nltk.download('punkt')
+nltk.download('averaged_perceptron_tagger')
+
```
Comment (BritVandy) [inline app/utils/complexity/squad_score.py:9]: Good catch. I always forget to add these at first.

**Review by BritVandy — APPROVED** [review]
So much easier than I was thinking it would be!

**Review by techthumb1 — APPROVED** [review]
Good quick and useful update to the file, good work!

**Review by Fellylove — APPROVED** [review]
Great work, Brittany!

**Pull request merged**